envoy-control-source-consul/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/consul/services/ConsulServiceMapper.kt

```diff
@@ -22,8 +20,8 @@ open class ConsulServiceMapper(
         return ServiceInstance(
             id = consulInstance.serviceId,
             tags = consulInstance.serviceTags.toSet(),
-            address = consulInstance.serviceAddress,
-            port = consulInstance.servicePort,
+            address = consulInstance.serviceAddress.orElse(""),
+            port = consulInstance.servicePort.orElse(0),
```

Comment:
i think it should be nullable,  imho port 0 is not equal to not present port

Reply:
changed